.circleci/config.yml

```diff
@@ -214,38 +214,38 @@ workflows:
           tag: 'deployed:dev'
           requires: [deploy_dev]
           context: [hmpps-common-vars]
-  #      - request-preprod-approval:
-  #          type: approval
-  #          requires:
-  #            - deploy_dev
-  #      - hmpps/deploy_env:
-  #          name: deploy_preprod
-  #          env: "preprod"
-  #          jira_update: true
-  #          jira_env_type: staging
-  #          context:
-  #            - hmpps-common-vars
-  #            - hmpps-accredited-programmes-ui-preprod
-  #          requires:
-  #            - request-preprod-approval
-  #          helm_timeout: 5m
-  #      - request-prod-approval:
-  #          type: approval
-  #          requires:
-  #            - deploy_preprod
-  #      - hmpps/deploy_env:
-  #          name: deploy_prod
-  #          env: "prod"
-  #          jira_update: true
-  #          jira_env_type: production
-  #          slack_notification: true
-  #          slack_channel_name: << pipeline.parameters.releases-slack-channel >>
-  #          context:
-  #            - hmpps-common-vars
-  #            - hmpps-accredited-programmes-ui-prod
-  #          requires:
-  #            - request-prod-approval
-  #          helm_timeout: 5m
```

Comment:
We don't want the `helm_timeout` values below?

Reply:
Turns out `5m` is the default value for `helm_timeout` according to the documentation [here](https://circleci.com/developer/orbs/orb/ministryofjustice/hmpps#jobs-deploy_env), so we don't need to set it, although I'm not sure what it does as there's no description provided...

I've followed the Temporary Accommodation/Approved Premises teams' approaches here.